bokeh/plotting/figure.py

```diff
@@ -181,7 +182,8 @@ def __init__(self, *arg, **kw):
 
         tool_objs, tool_map = process_tools_arg(self, opts.tools, opts.tooltips)
         self.add_tools(*tool_objs)
-        process_active_tools(self.toolbar, tool_map, opts.active_drag, opts.active_inspect, opts.active_scroll, opts.active_tap)
+        process_active_tools(self.toolbar, tool_map,
+            opts.active_drag, opts.active_inspect, opts.active_scroll, opts.active_tap, opts.active_multi)
```

Comment:
why is this line split? It's less than our line length of 165 FWIW I truly loathe this kind of splitting that leaves params at different vertical positions. Things I would suggest instead:
```
 some_func(foo, 
           bar,
           baz)

some_func(
    foo,
    bar,
    baz)

some_func(
    foo,
    bar,
    baz
)
```

or even:
```
some_func(
    foo, bar,
    baz
)
```
Any of those would be a drastic improvement over params starting in different places on different lines for the same function.

Reply:
Sure, although to me the only option is this:
```py
some_func(
    foo,
    bar,
    baz,
)
```
As to other options:
1. I commented earlier on this atrocity, which is something I despise, in this and other contexts (imports, data structures).
```py
some_func(foo, 
          bar,
          baz)
```
2. This makes two line diffs when adding arguments.
```py
some_func(
    foo,
    bar,
    baz)
```
3. Similarly here. We use trailing commas in other contexts, so we should use it here as well.
```py
some_func(
    foo,
    bar,
    baz
)
```

Reply:
@mattpap Great I think we are in agreement, I'm fine with 
```
some_func(
    foo,
    bar,
    baz,
)
``` 
since I agree it has the least amount of diff noise over time, so let's go with that. (It would be good to document this someplace)

Reply:
BTW I am also OK with everything on a single line, according to judgment. e.g. I would not split
```
foo(10, 20, "bar")
```
"just because"